### DISCUSSION TITLE
Feature Request: Background image support

### DISCUSSION THREAD

**Discussion by defunct73** [ghostty-org/ghostty#3639] [Feature Requests, Ideas] [answered]
Extremely low priority feature request, but would love to see support for a background image.

**Comment by mitchellh** [comment, marked as answer]
Approved. #3639

**Reply by fulopkovacs** [reply]
Related issue: https://github.com/ghostty-org/ghostty/issues/3645
(#3639 is this discussion)
(btw thanks for this feature in advance!)
